sympy/polys/densebasic.py

```diff
@@ -182,6 +182,36 @@ def dmp_degree(f, u):
     else:
         return len(f) - 1
 
+def dmp_power(f, u):
+    """
+    Return the powers of variable in ``f`` in ``x_0`` in ``K[X]``.
+
```

Comment:
Add a note on `Parameters`, `Returns` and `Raises`. Something like,
```
Parameters
=========
<your-docs>

Returns
======
<your-docs>

Raises
======
<your-docs>
```
Do this for all methods which are in the main namespace. Thanks. This is just a suggestion not a compulsion.

Reply:
This function `dmp_power(f, u)` is not that very important to be documented like you are suggesting. You can also refer to `dmp_degree` function. This is just a helper function to `power_list` and the documentation done is sufficient I think.

Reply:
resolve this one too. not needed